Configurateur/front/lib/screens/container-creation/container_creation/container_creation.dart

```diff
@@ -782,9 +808,9 @@ class ContainerCreationState extends State<ContainerCreation> {
                     const SizedBox(
```

Comment:
Maybe you can put the value of sizebox in the globalStyle and add it to all of the others